scalding-core/src/test/scala/com/twitter/scalding/typed/OptimizationRulesTest.scala

```diff
@@ -234,13 +240,13 @@ class OptimizationRulesTest extends FunSuite {
 
   test("all optimization rules don't change results") {
     import TypedPipeGen.{ genWithIterableSources, genRule }
-    implicit val generatorDrivenConfig: PropertyCheckConfiguration = PropertyCheckConfiguration(minSuccessful = 100000)
+    implicit val generatorDrivenConfig: PropertyCheckConfiguration = PropertyCheckConfiguration(minSuccessful = 500)
```

Comment:
did you mean to drop these to 500 in the commit/merge ?

Reply:
I pretty much hate scalatest. It was ignoring this parameter until I made this extend PropertyChecks. After I did this, 500 checks takes a while (200 seconds). I figured that would be enough for now.

Reply:
Yeah its a great exemplar of something with 2 many ways to do anything/everything and then its just too hard to do things right.

👍